eng/jacoco-test-coverage/pom.xml

```diff
@@ -84,7 +84,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>1.7.0-beta.3</version> <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
+      <version>2.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
```

Comment:
Just curious why this is in this PR instead of another PR about core-amqp.

Reply:
Once core-amqp is released , I will do another PR for for `core-amqp` version change.